### PR TITLE
Trading page typing calling getters

### PR DESCRIPTION
**Pull request by bthaile**
first piece in a long list of pieces: https://github.com/AugurProject/augur/issues/2457

 * fixed a few issues when lodash was removed. 
 * standardized selectedOutcome to selectedOutcomeId and typed as string
 * typed trading form, wrapper and the actual trading form components
 * fixed styling confusion between order form and common form.
 * changed marketType to be string on getMarketsInfo
   * added settlement fee
   * stubbed out reporting fee

### CODE REVIEW

File: packages/augur-ui/src/modules/market-charts/components/market-outcomes-chart/market-outcome-chart-highchart.tsx [hunk 1]
```diff
@@ -120,7 +134,7 @@ export default class MarketOutcomesChartHighchart extends Component {
             snap: true,
             label: {
               enabled: true,
-              format: "{value:.4f} <span class='eth-label'>ETH</span>"
+              format: "{value:.4f} <span class='eth-label'>DAI</span>"
```
Comment (justinbarry) [inline packages/augur-ui/src/modules/market-charts/components/market-outcomes-chart/market-outcome-chart-highchart.tsx:137]: Are we renaming classes to reflect the change? e.g. 'eth-label' ->'dai-label'.

Comment (bthaile) [reply packages/augur-ui/src/modules/market-charts/components/market-outcomes-chart/market-outcome-chart-highchart.tsx:137]: not at this time. We'll do that as part of the styling re-vamping

File: packages/augur-ui/src/modules/market-charts/components/market-outcome-charts--candlestick/market-outcome-charts-candlestick-highchart.tsx [hunk 2]
```diff
@@ -74,15 +74,15 @@ export default class MarketOutcomeChartsCandlestickHighchart extends Component {
               if (!props.isMobile) return;
               const { width } = this.renderer;
               this.ethLabel = this.renderer
-                .label("ETH", width - HighConfig.MobileEthlabelX, 0)
+                .label("DAI", width - HighConfig.MobileEthlabelX, 0)
```
Comment (andrewdamelio) [inline packages/augur-ui/src/modules/market-charts/components/market-outcome-charts--candlestick/market-outcome-charts-candlestick-highchart.tsx:77]: eventually, we will need to update the MobileEthlabelX var to be ```MobileDailabelX```

Same with ```potentialEthLoss``` and a few other vars -- we can clean up naming later

**Review by andrewdamelio — APPROVED** [review]
LGTM

**Review by justinbarry — APPROVED** [review]
lgtm

**Pull request merged**